scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,304 @@
+
+  #executing the sql 
+  $create_sql->execute();
+  $alter_sql->execute();
+
+  $create_sql->finish();
+  $alter_sql->finish();
+
+  #$create_backup_vf->execute();
+  #$create_backup_vf->finish();
+}
+
+sub create_merged_file {
+  # this would merge the files variation_feature.txt and variation_name_set.txt using the column they share in common which is the var_name and creates a new file 
+  # which is variation_name_set.txt containing the var_id and var_set_id which would be loaded to the new variation_set table
+  my $file = shift; 
+  my $second_file = shift;
+  my $third_file = shift;
+
+  open FILE1, "<", "$file" or die "Cannot open $file: $!";
+  open FILE2, "<", "$second_file" or die "Cannot open $second_file: $!";
+  open OUTPUT, ">", "$third_file" or die "Cannot open $third_file: $!";
+  my %data;
+
+  while (<FILE1>) {
+    chomp;
+    my $key = (split)[0]; # to access the variation name
+    my @value = (split)[1]; # to access set ids
+    # checking if the key exists in the dictionary and if it exists it appends the array reference
+    if (exists $data{$key}) {
+       push @{$data{$key}}, \@value; # push the array reference to the hash
+    } else { # if it does  not exist, it just creates a new key and an array
+      $data{$key} = [\@value];
+    }
+  }
+  while (<FILE2>) {
+    chomp;
+    my $var_name = (split)[1];
+    my $var_id = (split)[0];
+    if(exists $data{$var_name}) { # checking if the variation_name exists in the dictionary if it does 
+      my $array_ref = $data{$var_name}; # creating an array reference if the var_name exists in the created hash 
+      foreach my $set_id (@$array_ref) {
+        print OUTPUT "$var_id\t@$set_id[0]\n"; # to access the values in the array reference and the variation_id to create a new file
+      }
+    }
+  }
+  close FILE1;
+  close FILE2;
+  close OUTPUT;
+
+}
+
+sub load_all_variation_sets {
+  # this would load the created file from the function create_merged_file to a temp table that was created using temp_table function
+  my $dbhvar = shift;
+  my $load_file = shift;
+  
+  my $sql = qq{INSERT INTO temp_variation_set (variation_id, variation_set_id ) VALUES (?, ?)};
+  my $sth = $dbhvar->prepare($sql);
+
+  open FH, "<", "$load_file" or die "Can not open $load_file: $!";
+  while (<FH>) {
+    chomp;
+    my $var_id = (split)[0];
+    my $var_set_id = (split)[1];
+    
+    $sth->execute($var_id, $var_set_id);
+  }
+  close FH;
+  $sth->finish();
+
+}
+
+sub update_variation_feature_table {
+  # this function after populating the variation_feature_backup table created by inserting from the original table would then update the variation_set_id column uaing the file from the 
+  # dump_new_variation_sets
+  my $dbhvar = shift; 
+  my $load_file = shift;
+  
+  my $insert_temp_vf = $dbhvar->prepare(q{ INSERT into variation_feature_bk SELECT * from variation_feature });
+  
+  $insert_temp_vf->execute();
```

Comment:
This command fails:
`Table 'dlemos_homo_sapiens_variation_111_38.variation_feature_bk' doesn't exist at update_new_sets.pl line 189.`
The create command is a comment: line 102